frappe/public/js/frappe/ui/messages.js

```diff
@@ -128,6 +128,19 @@ frappe.msgprint = function(msg, title, is_minimizable) {
 		data.indicator = 'blue';
 	}
 
+	if (data.as_list) {
+		const list_rows = data.message.map(m => `<li>${m}</li>`).join('');
+		data.message = `<ul style="padding-left: 20px">${list_rows}</ul>`;
+	}
```

Comment:
do we require `as_list` flag? We can directly render message content as list if `data.message` is an array